### DISCUSSION TITLE
Fabric API won't allow me to load into a world

### DISCUSSION THREAD

**Discussion by mshailz** [FabricMC/fabric-api#2274] [Player support] [answered]
I'm currently on 1.18.2 and I installed the latest OptiFine along with the latest Fabric Loader, API, and OptiFabric. No mods besides OptiFine and Fabric. I can't load into my singleplayer worlds and I managed to narrow the issue down to it being Fabric API. I tried the Beta and Latest Release but nothing is working. Can someone please help me? I have the latest log here:
[latest.log](https://github.com/FabricMC/fabric/files/8818349/latest.log)

**Comment by Linguardium** [comment, marked as answer]
optifine is not compatible with the latest fabric api. downgrade it to 0.53.3 or earlier

**Reply by mshailz** [reply]
Would that be under 1.17.1 in CurseForge?

**Reply by Linguardium** [reply]
it would be in the files section on curseforge

**Reply by mshailz** [reply]
I apologize for my confusion; Is it Optifine that I have to downgrade, or Fabric API?

**Reply by Linguardium** [reply]
fabric api

**Reply by mshailz** [reply]
Found it, thank you very much!
